README.md

```diff
@@ -32,8 +32,20 @@ conn = Etcdv3.new(endpoints: 'https://hostname:port', user: 'root', password: 'm
 ```
 **High Availability**
 
-In the event of a failure, the client will work to restore connectivity by cycling through the specified endpoints until a connection can be established.  With that being said, it is encouraged to specify multiple endpoints when available.
+In the event of a failure, the client will work to restore connectivity by cycling through the specified endpoints until a connection can be established.  With that being said, it is encouraged to specify multiple endpoints when available. That's the default
+behaviour.
 
+However, sometimes this is not what you want. If you need more control over
+failures (and therefore having `timeout:` provide at least some guarantees),
```

Comment:
I would remove `(and therefore having `timeout:` provide at least some guarantees)`.  It doesn't provide any value in this context.  `timeout:`  provides guarantees surrounding the duration of the actual command execution, so this statement isn't really accurate.  

If you want to improve or help clarify the `timeout` flag we should be doing it here.  https://github.com/davissp14/etcdv3-ruby#timeouts